account/identity.go

```diff
@@ -138,6 +142,24 @@ func (m *GormIdentityRepository) Load(ctx context.Context, id uuid.UUID) (*Ident
 	return &native, errs.WithStack(err)
 }
 
+// Exists returns true|false where an object exists with an identifier
+func (m *GormIdentityRepository) Exists(ctx context.Context, id uuid.UUID) (bool, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "identity", "exists"}, time.Now())
+	var exists bool
+	query := fmt.Sprintf(`
+		SELECT EXISTS (
+			SELECT 1 FROM %[1]s
+			WHERE
+				id=$1
+				AND deleted_at IS NULL
+		)`, m.TableName())
```

Comment:
yeah! I had that in mind I just want to be more explicit to show the cases... I think the tracker and some others have few differences cause they include space id, etc... But generally, that was the code polishing part to be done. Thx @kwk